client/app/components/BottomTabs.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import styles, { colors } from '../config/styles';
+import SimpleLineIcons from '@expo/vector-icons/SimpleLineIcons';
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+class BottomTabs extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    tabButtons = [
+        {
+          viewName: 'FriendsView',
+          IconComponent: SimpleLineIcons,
+          iconName: 'people',
+          opacity: this.props.viewName == 'FriendsView' ? 1 : .3,
```

Comment:
Do we want viewName to be a part of this element's props? It feels more appropriate to make it app level state.

Reply:
I thought about doing it like that, and we could, but sometimes we aren't on one of those three views. For example, say you click "berkeley" and now you see a menu for the day on your screen. You're not really still "on" any of the tabs - hitting the menus tab will change your view and bring you back to the list of dining halls screen. We could clear the redux state every time you like change your view, and then have the tabs component read from the redux state to figure out which icon to highlight (or none of them), but it seemed simpler to have you pass in the viewname as props when the view includes the tabs component. Either way should work though if we think I should change it

Reply:
Specifically, I'm thinking we need this to restore state when the user hits the 'back' button. We could always store that in a stack elsewhere in the state, but I think it might be a good idea to do here. @mooreorless555 do you have thoughts?